docs: Update README.md (#4)
clightning4j/kotlin-template: README.md

# kotlin-template

> A minimal template for projects based on [JRPClightning](https://github.com/vincenzopalazzo/JRPClightning). This should get you started if you want to develop a c-lightning plugin using Java.

Reviewer: ```suggestion
A minimal template for projects based on [JRPClightning](https://github.com/vincenzopalazzo/JRPClightning). This should get you started if you want to develop a core lightning plugin using Java.
```

> A minimal template for projects based on [JRPClightning](https://github.com/vincenzopalazzo/JRPClightning). This should get you started if you want to develop a core lightning plugin using Java.

Reviewer: ```suggestion
A minimal template for projects based on [JRPClightning](https://github.com/vincenzopalazzo/JRPClightning). This should get you started if you want to develop a core lightning plugin using Kotlin.
```

Sorry I missed it before

## Install

> Java produces a jar and c-lightning needs a bash script to run it!

Reviewer: ```suggestion
Java produces a jar and core lightning needs a bash script to run it!
```

> The gradle script compiles the plugin and generate a bash script using:

Reviewer: ```suggestion
The gradle is able to compile the plugin in a jar, and generate a runnable script that core lightning use to run it
```

```bash

> ./gradlew formatKotlin

Reviewer: ```suggestion
./gradlew createRunnableScript
```

```

After the gradle process, you will have the jar inside the `build/libs/app-all.jar` and the script `app-gen.sh`
in the `app` directory of the project.

Now you can put the gradle script inside the c-lightning plugin directory or use the `--plugin="PATH/TO/THE/app-gen.sh`.

## Code Style
[![ktlint](https://img.shields.io/badge/code%20style-%E2%9D%A4-FF4081.svg)](https://ktlint.github.io/)

> We live in a world where robots can drive a car, so we shouldn't just write code, we should write elegant code.

This repository use [ktlint](https://github.com/pinterest/ktlint) to maintains the code of the repository elegant, so
before submit the code check the Kotlin format with the following command on the root of the directory

```bash
./gradlew formatKotlin
```

## Built with

- [JRPClightning](https://github.com/vincenzopalazzo/JRPClightning)

## Support
If you like the library and want to support it, please considerer to donate with the following system

> 

Reviewer: ```suggestion
```

- [liberapay.com/vincenzopalazzo](https://liberapay.com/vincenzopalazzo)
- [3BQ8qbn8hLdmBKEjt1Hj1Z6SiDsnjJurfU](bitcoin:3BQ8qbn8hLdmBKEjt1Hj1Z6SiDsnjJurfU)
- [Github support](https://github.com/sponsors/vincenzopalazzo)

## License

<div align="center">
  <img src="https://opensource.org/files/osi_keyhole_300X300_90ppi_0.png" width="150" height="150"/>
</div>

> It is a c-lightning plugin to override Bitcoin backend plugin with esplora.

Reviewer: ```suggestion
It is a core lightning plugin to override Bitcoin backend plugin with esplora.
```

Reviewer: Maybe it is better

>It is a core lightning plugin template to develop plugin with Kotlin JVM

> Copyright (C) 2020-2021 Vincenzo Palazzo vincenzopalazzodev@gmail.com

Reviewer: ```suggestion
Copyright (C) 2020-2022 Vincenzo Palazzo vincenzopalazzodev@gmail.com
``
